docs/tasks/run-as-deployment/quickdockerbuildusingidentitykeyvault/README.md

```diff
@@ -0,0 +1,74 @@
+# Quick Docker build using identity and keyvault
+
+## Create a resource group
+
+```bash
+az group create \
+  -n mytaskrunrg \
+  -l westus
+```
+
+## Create a Registry
+
+```bash
+az acr create \
+   -n myreg -g mytaskrunrg --sku Standard
+```
+
+## Create a Custom Registry
+
+```bash
+az acr create \
+   -n mycustomreg -g mytaskrunrg --sku Standard
+```
+
+## Create a User Identity
+
+```bash
+az identity create \
+  -g mytaskrunrg \
+  -n myquickdockerbuildrunwithidentity
+```
+
+## Create KeyVault
+
+```bash
+az keyvault create --name mykeyvault --resource-group mytaskrunrg --location eastus2
+```
+
+## Save registry username/password in the keyvault
+
+```bash
+az keyvault secret set --name username --value <username> --vault-name mykeyvault
+az keyvault secret set --name password --value <password> --vault-name mykeyvault
+```
+
+## Grant identity access to key vault (object-id is the Object ID of managed identity)
+```bash
+az keyvault set-policy --name mykeyvault --resource-group mytaskrunrg --object-id 452e1d96-c423-4da0-99f2-d3a1789ab69f --secret-permissions get 
```

Comment:
>452e1d96-c423-4da0-99f2-d3a1789ab69f [](start = 82, length = 36)

can you get the object id from identity command?

Reply:
fixed.